### ISSUE TITLE
Debug button not working

### ISSUE THREAD

**Issue by IcyTotem** [hbenl/vscode-mocha-test-adapter#65]
I am using mocha explorer in a TypeScript project. Everything worked fine up until two days ago when suddenly the debug test button stopped working, meaning that upon click it simply does nothing. I can run all tests just fine. I can also run single tests fine, but I cannot debug them. There are no messages or errors in the logs. I didn't change the configuration and I didn't install new plugins. I tried to reinstall MochaExplorer but the result was the same. I can still debug just fine using a VSCode mocha task, but I have to specify the exact path of the test file I want to debug.

These are my vs code settings:

```
"mochaExplorer.cwd": "application/core",
    "mochaExplorer.mochaPath": "application/core/node_modules/mocha",
    "mochaExplorer.files": "application/core/test/**/*.ts",
    "mochaExplorer.logpanel": true,
    "mochaExplorer.require": "ts-node/register",
    "mochaExplorer.env": {
        ...
    }
```

I also tried to override the debugConfig with a copy of the default one, but nothing happened. It looks like the button is simply dead and does not respond to anything. I don't know where to look to find any clue.

**Comment by hbenl** [comment]
Please have a look which version of Test Explorer you're running. It's probably 2.14.4, which introduced this bug. I have published a fix in version 2.14.5 a couple of hours ago.

**Comment by IcyTotem** [comment]
I can confirm updating to 2.14.5 solves the issue, thank you.
